### ISSUE TITLE
Learnable Boundary Guided Adversarial Training 

### ISSUE THREAD

**Issue by jiequancui** [robust-ml/robust-ml.github.io#18]
**Name**: {Learnable Boundary Guided Adversarial Training }

**Authors**: {Jiequan Cui, Shu Liu, Liwei Wang, Jiaya Jia}

**Paper**: {https://arxiv.org/abs/2011.11164}

**Code**: [code](https://github.com/FPNAS/LBGAT)

**Venue**: {venue if published}

**Does the code implement the robust-ml API and include pre-trained models**: {yes}

**Dataset**: {CIFAR-100}

**Threat model**: {threat model for which claims are made}

**Natural accuracy**: {60.55}

**Claims**: {30.20}

**Comment by anishathalye** [comment]
I don't see the implementation of the robust-ml API. Am I looking in the wrong place?

**Comment by jiequancui** [comment]
Hi, are there some examples for PyTorch to implement with robust-ml API?

**Comment by anishathalye** [comment]
You can see a bunch of examples here: https://www.robust-ml.org/defenses/. Here is one that uses torch: https://github.com/bethgelab/AnalysisBySynthesis. The robust-ml API is independent of the ML library you use, so it shouldn't really make much of a difference between tensorflow and torch etc.
